templates/page_object.rb

```diff
@@ -0,0 +1,7 @@
+class PageObject
```

Comment:
This seems cool, but I strongly believe that Suspenders should contain configuration as much as possible, and we should limit the amount of code we copy into generated apps.

We don't use the page object pattern on every application, and I think it would make sense to verify that we want this to be our standard approach before including it.

Could this code form the base of a library for building tests from page objects?